src/kv/value.rs

```diff
@@ -765,13 +596,586 @@ where
+            Inner::None => visitor.visit_null(),
+            Inner::Bool(v) => visitor.visit_bool(*v),
+            Inner::Str(v) => visitor.visit_borrowed_str(*v),
+            Inner::Char(v) => visitor.visit_char(*v),
+            Inner::I64(v) => visitor.visit_i64(*v),
+            Inner::U64(v) => visitor.visit_u64(*v),
+            Inner::F64(v) => visitor.visit_f64(*v),
+            Inner::I128(v) => visitor.visit_i128(*v),
+            Inner::U128(v) => visitor.visit_u128(*v),
+            Inner::Debug(v) => visitor.visit_any(Value::from_dyn_debug(*v)),
+            Inner::Display(v) => visitor.visit_any(Value::from_dyn_display(*v)),
+        }
+    }
+}
+
+impl<'v> Value<'v> {
+    /// Get a value from a type implementing `std::fmt::Debug`.
+    #[cfg(feature = "kv_unstable")]
+    #[deprecated(note = "use `from_debug` instead")]
+    pub fn capture_debug<T>(value: &'v T) -> Self
+    where
+        T: fmt::Debug + 'static,
+    {
+        Value::from_debug(value)
+    }
+
+    /// Get a value from a type implementing `std::fmt::Display`.
+    #[cfg(feature = "kv_unstable")]
+    #[deprecated(note = "use `from_display` instead")]
+    pub fn capture_display<T>(value: &'v T) -> Self
+    where
+        T: fmt::Display + 'static,
+    {
+        Value::from_display(value)
+    }
+
+    /// Get a value from an error.
+    #[cfg(feature = "kv_unstable_std")]
+    #[deprecated(note = "use `from_dyn_error` instead")]
+    pub fn capture_error<T>(err: &'v T) -> Self
+    where
+        T: std::error::Error + 'static,
+    {
+        Value::from_dyn_error(err)
+    }
+
+    /// Get a value from a type implementing `serde::Serialize`.
+    #[cfg(feature = "kv_unstable_serde")]
+    #[deprecated(note = "use `from_serde` instead")]
+    pub fn capture_serde<T>(value: &'v T) -> Self
+    where
+        T: serde::Serialize + 'static,
+    {
+        Value::from_serde(value)
+    }
+
+    /// Get a value from a type implementing `sval::Value`.
+    #[cfg(feature = "kv_unstable_sval")]
+    #[deprecated(note = "use `from_sval` instead")]
+    pub fn capture_sval<T>(value: &'v T) -> Self
+    where
+        T: sval::Value + 'static,
+    {
+        Value::from_sval(value)
+    }
+
+    /// Check whether this value can be downcast to `T`.
+    #[cfg(feature = "kv_unstable")]
+    #[deprecated(
+        note = "downcasting has been removed; log an issue at https://github.com/rust-lang/log/issues if this is something you rely on"
+    )]
+    pub fn is<T: 'static>(&self) -> bool {
+        false
+    }
+
+    /// Try downcast this value to `T`.
+    #[cfg(feature = "kv_unstable")]
+    #[deprecated(
+        note = "downcasting has been removed; log an issue at https://github.com/rust-lang/log/issues if this is something you rely on"
+    )]
+    pub fn downcast_ref<T: 'static>(&self) -> Option<&T> {
+        None
+    }
+}
+
+// NOTE: Deprecated; but aliases can't carry this attribute
+#[cfg(feature = "kv_unstable")]
+pub use VisitValue as Visitor;
```

Comment:
Should this be visit?
```suggestion
pub use VisitValue as Visit;
```